users_ldap_push/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+==================
+Push users to LDAP
+==================
+
+This module was written in order to use Odoo as a frontend for creating LDAP
+entries by creating user records. Updates to the user record will be propagated
+to the linked LDAP entry afterwards.
+
+When users change their passwords, they will be updated in the LDAP directory
+too.
+
+Configuration
+=============
+
+On the LDAP parameters of your company, check *Create ldap entry* in order to
+activate this functionality. Be sure to configure a bind DN that has
+appropriate permissions to create and modify entries.
+
+Fill in the object classes newly created entries should contain, separated by
+colons. Those classes will determine which mappings from Odoo fields to LDAP
+attributes you need. This is highly dependent on your LDAP setup.
+
+For a standard slapd setup, you might want to use object classes
+`inetOrgPerson,shadowAccount` and the following mapping:
+
+========== ============== ==
+Odoo field LDAP attribute DN
+========== ============== ==
+Login      userid         X
+Name       cn
+Name       sn
+========== ============== ==
+
+Matching is done by the new field *ldap_entry_dn*, so after installing this
+module, you'll probably want to set this field. The module will write it when
+a user logs in via Odoo.
+
+Usage
+=====
+
```

Comment:
From reading the code, I gather that a checkbox in the user form is used to decide at the time of user creation if the user should be stored in LDAP. If this is the case, can you add this feature to this section?